CASSANDRA.md

```diff
@@ -307,9 +300,34 @@ object Phantom extends App {
 }
 ```
 
-Phantom allows the user certain level of composability, but it gets a bit verbose due to the nature of the DSL. 
+Phantom allows the user certain level of composability, but it gets a bit verbose due to the nature of the DSL.
+However, phantom does not encourage this inherent duplication of concern because it doesn't add any value
+to an application, and instead generally prefers to duplicate from the start.
+
+On a character count, this is likely to be the shortest approach of all frameworks.
```

Comment:
@fwbrasil Ok fair enough, I can definitely say that's extremely true for SQL, I guess ultimately it's the simple nature of CQL that doesn't make this feature that much more interesting when it comes to Cassandra. I'm happy to remove that line, I was pursuing the very practical benefit and I missed the theory.